### ISSUE TITLE
Relationship between 4.A and 6.B

### ISSUE THREAD

**Issue by AnnMarieDavis** [Section508Coordinators/DHSA-TT-200-A#307]
## Issue Type
- [ ] System Bug (the course doesn't function properly)
- [ ] Typo/grammar
- [ ] Incorrect content
- [ ] 508 issue (something is not accessible according to our own test process)
- [X] Other, specify...Clarification for students

## Description
If a link displays content, it should be tested for 6.A even if the revealed content was not keyboard accessible. 

## Expected Behavior / Recommendation
Add to Chapter 6, Lesson 2 as a note to students
